packages/eas-build-job/src/ios.ts

```diff
@@ -125,6 +136,9 @@ export const JobSchema = Joi.object({
   cache: CacheSchema.default(),
   developmentClient: Joi.boolean(),
   simulator: Joi.boolean(),
+  version: Joi.object({
+    buildNumber: Joi.string(),
```

Comment:
No validation for this field? Should it be possible to pass an empty string here?

Reply:
joi does not allow empty strings by default